wiki/Article_Style_Guide/en.md

```diff
@@ -1,627 +1,468 @@
+There are two markup styles for links:
 
 ```
-/wiki/shared/False.png
-```
+inline:
 
-Use of this type of link may include:
+[example](https://example.com/)
 
-- linking to an article within the osu!wiki.
-- linking to a shared image (images that are used in multiple places).
+---
 
-#### Relative-relative Links
+reference:
 
-Relative-relative links are relative links that start from the current directory.
-For example:
+[wikilink-welcome]: /wiki/Welcome
 
-```
-img/example.png
-./img/example.png
-../Insane/
+[example][wikilink-welcome]
 ```
 
-Use of this type of link may include:
-
-- linking to an article that is a subdirectory of an article.
-- linking to images for a specific article.
+You can use either the reference or inline style links.
 
-### Link Text
+If you are using the reference style linking, it is suggested to place the reference links at the top of the article for quicker access.
 
-Do **not** use the word `here` as the link text.
-As it may seem convenient to use, this may cause confusion and/or be misleading.
+#### Internal
 
-Instead, take the page's title (from the top of page or from the `<title/>` tag) and use that as the link text.
-If there is not a viable or obvious title, then use the site's name or a general description instead.
+Internal links are the links linking within the osu!wiki.
 
-### Section Links
+Do not use the word `here` as the link text.
+Instead, take the title of the page the link is linking to or use a general description.
 
-All section links follow these strict rules:
+All internal links are to use redirects.
+To do this, open the [`redirect.yaml`](https://github.com/ppy/osu-wiki/blob/master/wiki/redirect.yaml) file.
+The link part that you want is on the left while the designation to those link parts are on the right.
+Keep note of the redirect you want to use.
+Then make the link like you normally would, but instead of using the directory path, use `/wiki/` followed by the redirect.
 
-- all letters **must** use lowercase letters
-- all special characters (e.g. `!`, `?`, `,`, `"`, `'`) **must** be replaced with a hyphen (`-`)
-  - if one of the special characters is the last character in the section name, remove it, but do not add a hyphen (`-`).
-- all spaces are replaced with hyphens (`-`), not underscores (`_`)
-- if a header is not unique (two or more of the same section heading names exist at any level):
-  - the first instance of said heading will follow the above rules
-  - the following instances **must** follow the above rules **and** must add `.` followed by an incremental integer starting from `2`
-- if a section contains an image, ignore it
-- if a section starts with a space, ignore it
+**Examples:**
 
-Notice: you will not be able to check if section link links to the correct location in some scenarios.
+- `[ASG](/wiki/ASG)`
+- `[OWC2015](/wiki/OWC2015)`
+- `[skinning tutorial](/wiki/skinning_tutorial)`
 
-Examples:
+##### Why are we using redirects instead of paths?
```

Comment:
Please explain why this change should happen. I will not change it unless you provide a good explanation of some sort.